### DISCUSSION TITLE
【羊村】重金悬赏项目

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/home#245] [Show and tell]
- 本讨论帖为：重金悬赏项目，刚兴趣的来认领！

**Comment by jiangzhonglian** [comment]
请问哪位有申请过m6doc中文版面数据集（官方要求是近6年发表过ocr、笔迹分析、文档图像处理等方面的论文）

- 地址：https://github.com/hciilab/m6doc
- 联系方式：qq 838911591
- 如果可以分享的话，有偿5k。

**Comment by wizardforcel** [comment]
https://academic.chatwithpaper.org/

学术头条计划：我们打算用ChatGPT做一个纯中文的Arxiv，让所有不懂英文的人也能接触前沿的计算机和AI技术，预算1000，联系🐧1985790413，或加群582943080。

**Comment by jiangzhonglian** [comment]
> 组织悬赏任务

- 长期可以做的翻译/贡献的项目
- https://github.com/orgs/apachecn/discussions/243

**Comment by jiangzhonglian** [comment]
1
